.circleci/config.yml

```diff
@@ -67,16 +54,35 @@ workflows:
               ignore: /.*/
             tags:
               only: '/^v[\d.]+$/'
+  nightly:
+    triggers:
+      - schedule:
+          cron: 0 7 * * *
+          filters:
+            branches:
+              only: master
+    jobs: *workflow_jobs
```

Comment:
Instead of doing `remove_package_lock` as a step inside `unit_tests_steps`, is there a way to run a single step or another job that's just for "setting up nightly tests" before `workflow_jobs` starts?

Reply:
Not sure that it's possible, because removing `package-lock.json` should happen in each job, as each job is run in a separate docker container and checks out sources separately. Technically we can run node8 + lint + docs + system tests + samples test in one container that is set up just once but it will make finding failures harder.